### ISSUE TITLE
Is this working?

### ISSUE THREAD

**Issue by martijnrusschen** [csstaub/homebridge-canary#1]
Would love to have this.

**Comment by csstaub** [comment]
Yeah, I did use this for a while. I ultimately moved away from Canary though so I haven't been updating this repo anymore. It may or may not work anymore at this point.
